pkg/cmd/server/api/v1/conversions.go

```diff
@@ -128,6 +128,14 @@ func init() {
 				obj.MCSLabelsPerProject = 5
 			}
 		},
+		func(obj *IdentityProvider) {
+			if len(obj.MappingMethod) == 0 {
+				// By default, only let one identity provider authenticate a particular user
+				// If multiple identity providers collide, the second one in will fail to auth
+				// The admin can set this to "add" if they want to allow new identities to join existing users
+				obj.MappingMethod = "claim"
```

Comment:
@smarterclayton This is breaking change.  The current default is `add`.  While I do think that `claim` is a better default, this means that old `master-config.yaml`s will suddenly start behaving differently after an upgrade: something we previously said we wouldn't do.

We could make this defaulter choose `add` and generate new configs using `claim` to avoid the break, but have better defaults moving forward.